ts/src/local/conductor.ts

```diff
@@ -220,26 +220,17 @@ export class Conductor implements IConductor {
 
     const startPromise = new Promise<void>((resolve) => {
       runConductorProcess.stdout.on("data", (data: Buffer) => {
+        logger.info(data.toString());
         const conductorLaunched = data
           .toString()
           .match(/Conductor launched #!\d ({.*})/);
-        const holochainRunning = data
-          .toString()
-          .includes("Connected successfully to a running holochain");
```

Comment:
Interesting, I think this is a sandbox line we were looking for. So some change in the sandbox caused this issue?

Looks like a good fix because it's simpler so yay. Just curious

Reply:
That code where the logs are output is 2 years old, so I think it's not a sandbox change. The thing is that usually when you look at the outputs of starting a sandbox, `Connected successfully to a running holochain" is output first, followed by `Conductor launched`. But in the code it's the other way around, `Conductor launched` is in fact the last line.

I don't know why this hasn't been creeping up earlier. Maybe no one paid attention to it too much?

On another note, this connected successfully log message makes no sense to me (anyone?), I'd like to remove it from the sandbox startup.

Reply:
Oh this woke something in my memory. I moved those lines around in January, so this is my mistake I'm afraid. Yes that log line makes very little sense. It's really saying "We saved the lock file to remember which running conductors we have". We were logging that the conductor was ready to use before saving that lock file though which I thought was weird so I swapped the log lines.

I had no idea they were being read downstream.

Reply:
https://github.com/holochain/holochain/pull/4587